### PR TITLE
Completes fetching and displaying the article list

### PR DESCRIPTION
**Pull request by jcainuk**
Completes fetching and displaying the article list as in step 4. The buttons will link up to the full article with image and description later

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,35 +1,17 @@
...
-      </p>
-    </>
-  )
-}
+    <div className="container">
+      <Routes>
+        <Route exact path="/" element={<Home />} />
+        <Route exact path="/articles/:id" element={<ArticleDetailPage />} />
+      </Routes>
+    </div>
```
Comment (Harrison365) [inline src/App.jsx:13]: Nice set up for routes 👍

Comment (jcainuk) [reply src/App.jsx:13]: Thank you :)

File: src/components/ArticleList.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await ArticleFinder.get("/");
+        setArticles(response.data.articles);
+      } catch (err) {
+        console.log(err);
+      }
+    };
+
```
Comment (Harrison365) [inline src/components/ArticleList.jsx:18]: Nice useEffect and api request 👍

File: src/components/ArticleList.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+              <tr key={`${article.title}+${article.index}`}>
+                <td>{article.title}</td>
+                <td>{article.topic}</td>
+                <td>{article.author}</td>
+                <td>{convertDate(article.created_at)}</td>
+                <td>{article.votes}</td>
+                <td>
+                  <button className="btn btn-warning">View article</button>
+                </td>
+              </tr>
```
Comment (Harrison365) [inline src/components/ArticleList.jsx:47]: Nice map 👍

**Pull request merged**